cohere/responses/chat.py

```diff
@@ -1,25 +1,31 @@
 import json
+from enum import Enum
 from typing import Any, Dict, Generator, List, NamedTuple, Optional
 
 import requests
 
 from cohere.responses.base import CohereObject
 
+Mode = Enum("Mode", ["chat", "search_query_generation", "augmented_generation"])
```

Comment:
i think enum values are normally all caps

Reply:
Have never really used Enum, but after testing around a bit, this doesn't quite seem what you want?
```
Mode = Enum("Mode", ["chat", "search_query_generation", "augmented_generation"])
Mode.chat=='chat' # False
Mode.chat.value == 1 # True
```